src/vl-upload.js

```diff
@@ -158,10 +164,10 @@ export class VlUpload extends vlFormValidationElement(vlElement(HTMLElement)) {
             <button type="button" class="vl-upload__element__button vl-link">
               <i class="vl-vi vl-vi-paperclip" aria-hidden="true"></i>
               <span class="vl-upload__element__button__container"></span>
-              <slot id="title-slot" name="title"></slot>
+              <span class="vl-upload__element__button__container"><slot name="title"></slot></span>
```

Comment:
Ga je  op deze manier geen styling issues krijgen in bepaalde gevallen (2 spans).

Reply:
De 1e span wordt verwijderd als er een slot aanwezig is, zie `processSlots`.

Reply:
Ja, dat had ik gezien, maar omgekeerd toch niet?

Reply:
Klopt, dan kunnen we voor de zekerheid best die logica ook nog toevoegen.